tests/unit/test_read_config.py

```diff
@@ -227,14 +227,94 @@ def test_010_WRF_NCI_config_object(config_path_wrf_docker):
     assert config == asdict(wrf_config)
 
 
-def test_011_valid_CMAQ_NCI_config_file(config_path_cmaq_nci):
-    setup_cmaq = load_cmaq_config(config_path_cmaq_nci)
-    assert setup_cmaq
+@pytest.fixture
+def dynamic_wrf_nci_values():
+    return [
```

Comment:
Some values in the WRF configuration are dependent on the environment variables, e.g. `${HOME}/..` Those values will be different in the local environment and the CI, so I removed them.

Reply:
Nice. I had to play lots of tricks to handle that correctly in openmethane